### PR TITLE
Np batch snm3c PD-2482

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

This pr includes the merging of several tasks: 
- Separate_unmapped_reads & Split_unmapped_reads
-  Merge_original_and_split_bam_and_sort_all_reads_by_name_and_position & dedup_unique_bam_and_index_unique_bam
- Unique_reads_allc & Unique_reads_cgn_extraction
- Hisat_single_end_r1_r2_mapping_dna_mode_and_merge_sort_split_reads_by_name & Remove_overlap_read_parts

This pr removed several outputs of the pipeline. @kayleemathews does this pr need doc updates?

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/snM3C/snM3C.wdl
Comment (ekiernan) [inline pipelines/skylab/snM3C/snM3C.wdl:126]: Are there files being tarred here that shouldn't be included? Should we just output the two file types that they are looking for? This might be a question for Brian too.

**Review by kevinpalis — APPROVED** [review]
LGTM

**Review by ekiernan — APPROVED** [review]
Thanks @nikellepetrillo ! LGTM as long as summary file matches after changes. I had just one question about the contact summary tar.

**Review by khajoue2 — APPROVED** [review]

File: pipelines/skylab/snM3C/snM3C.wdl
Comment (nikellepetrillo) [reply pipelines/skylab/snM3C/snM3C.wdl:126]: So i opened up this chromatin_contact_stats tar file, and it is filled with files ending in `hisat3n_dna.all_reads.contact_stats.csv `. This is not on Wubin's output list. 

It looks like he wants `.hisat3n_dna.all_reads.3C.contact.tsv.gz` and `.hisat3n_dna.all_reads.dedup_contacts.tsv.gz`. I will add these 2 outputs as final outputs. 

It does look like we still need the `hisat3n_dna.all_reads.contact_stats.csv` files as it is input to the summary task, so i will keep that one but will not set it as a final output. Let me know if this makes sense to you!

**Review by kayleemathews — APPROVED** [review]
@nikellepetrillo Docs are updated! If you can review them before you merge that would be great!

**Comment by nikellepetrillo** [comment]
retest this please

File: pipelines/skylab/snM3C/snM3C.wdl
Comment (ekiernan) [reply pipelines/skylab/snM3C/snM3C.wdl:126]: This plan sounds good. Thank you.

**Pull request merged**
